guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -1,41 +1,38 @@
+** `view_authenticators`, `create_authenticators`, `edit_authenticators`
+** `view_locations`, `assign_locations`
+** `view_organizations`, `assign_organizations`
 
 .Procedure
-. Set the Network Information System (NIS) service boolean to true to prevent SELinux from stopping outgoing LDAP connections:
+. On your {ProjectServer}, set the Network Information System (NIS) service boolean to true to prevent SELinux from stopping outgoing LDAP connections:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # setsebool -P nis_enabled on
 ----
 . In the {ProjectWebUI}, navigate to *Administer* > *Authentication Sources*.
-. Click *Create LDAP Authentication Source*.
-. On the *LDAP server* tab, enter the LDAP server's name, host name, port, and server type.
-The default port is 389, the default server type is POSIX (alternatively you can select FreeIPA or Active Directory depending on the type of authentication server).
-For `TLS` encrypted connections, select the *LDAPS* checkbox to enable encryption.
-The port should change to 636, which is the default for LDAPS.
-. On the *Account* tab, enter the account information and domain name details.
-See xref:Description_of_LDAP_Settings_{context}[] for descriptions and examples.
-. On the *Attribute mappings* tab, map LDAP attributes to {Project} attributes.
-You can map login name, first name, last name, email address, and photo attributes.
-See xref:Example_Settings_for_LDAP_Connections_{context}[] for examples.
-. On the *Locations* tab, select locations from the left table.
-Selected locations are assigned to users created from the LDAP authentication source, and available after their first login.
-. On the *Organizations* tab, select organizations from the left table.
-Selected organizations are assigned to users created from the LDAP authentication source, and available after their first login.
-. Click *Submit*.
-. Configure new accounts for LDAP users:
-+
-* If you did not select *Automatically Create Accounts In {Project}* checkbox, see {AdministeringDocURL}Creating_a_User_admin[Creating a User] in _{AdministeringDocTitle}_ to create user accounts manually.
+. From the *LDAP* menu, select *Create*.
+.. On the *LDAP server* tab, enter the details of your LDAP server.
 +
-* If you selected the *Automatically Create Accounts In {Project}* checkbox, LDAP users can now log in to {Project} using their LDAP accounts and passwords.
-After they log in for the first time, the {Project} administrator has to assign roles to them manually.
-For more information on assigning user accounts appropriate roles in {Project}, see {AdministeringDocURL}Assigning_Roles_to_a_User_admin[Assigning Roles to a User] in _{AdministeringDocTitle}_.
+For TLS encrypted connections, select *LDAPS* to enable encryption.
+Selecting *LDAPS* changes the default port in {ProjectWebUI} to `636`, which is the default for LDAPS.
+.. On the *Account* tab, enter the account information and domain name details.
```

Comment:
I am not sure whether the user will see the examples in the sections after this procedure. You might want to restore the xrefs.